### ISSUE TITLE
Resize(Start or End) Appointment in Scheduler

### ISSUE THREAD

**Issue by KevinKrueger** [radzenhq/radzen-blazor#1833]
What not only I need is that the appointments can be touched at the start or end and then dragged into length. When dragging, a kind of preview appears and automatically snaps into the grid. As soon as this is released, the date-time value for start or end is adopted, depending on where you grab the appointment with the mouse.
The mouse pointer must also be displayed accordingly. Left-Right, Top-Down (Arrows) depending on the view.

**Comment by KevinKrueger** [comment]
What do you think? Could you implement this promptly?

I have a bit of code that already adjusts the cursor, but for the time being it can only be left-right and drag the appointment, you will certainly have to rework it.

**Comment by akorchev** [comment]
> What do you think? Could you implement this promptly?

I am afraid not. This isn't a trivial feature to implement and support in all views.

**Comment by KevinKrueger** [comment]
> I am afraid not. This isn't a trivial feature to implement and support in all views.

Okay but can i give you the code i wrote that implements the first beginnings of resizing?

**Comment by akorchev** [comment]
We prefer not to include an incomplete solution.

**Comment by KevinKrueger** [comment]
Yes, of course, I'm more interested in you continuing the beginning because it would take me ages to understand the complete code. I didn't mean a pull request but to upload the file here in Markdown.
